hushline/routes.py

```diff
@@ -51,6 +62,13 @@ def get_directory_usernames() -> Sequence[Username]:
     ).all()
 
 
+def directory() -> str:
```

Comment:
What does this function do? In the routes it's being overwritten by another `def directory()`? It looks like it's fully unused.

Reply:
good catch. yes, looks like a dupe.

Reply:
c28c55a